Gemfile.lock

```diff
@@ -55,52 +55,52 @@ GEM
       ransack (~> 2.1, >= 2.1.1)
       sassc-rails (~> 2.1)
       sprockets (>= 3.0, < 4.1)
-    activejob (6.0.2.2)
-      activesupport (= 6.0.2.2)
+    activejob (6.0.3.1)
+      activesupport (= 6.0.3.1)
       globalid (>= 0.3.6)
-    activemodel (6.0.2.2)
-      activesupport (= 6.0.2.2)
-    activerecord (6.0.2.2)
-      activemodel (= 6.0.2.2)
-      activesupport (= 6.0.2.2)
-    activestorage (6.0.2.2)
-      actionpack (= 6.0.2.2)
-      activejob (= 6.0.2.2)
-      activerecord (= 6.0.2.2)
+    activemodel (6.0.3.1)
+      activesupport (= 6.0.3.1)
+    activerecord (6.0.3.1)
+      activemodel (= 6.0.3.1)
+      activesupport (= 6.0.3.1)
+    activestorage (6.0.3.1)
+      actionpack (= 6.0.3.1)
+      activejob (= 6.0.3.1)
+      activerecord (= 6.0.3.1)
       marcel (~> 0.3.1)
-    activesupport (6.0.2.2)
+    activesupport (6.0.3.1)
       concurrent-ruby (~> 1.0, >= 1.0.2)
       i18n (>= 0.7, < 2)
       minitest (~> 5.1)
       tzinfo (~> 1.1)
-      zeitwerk (~> 2.2)
+      zeitwerk (~> 2.2, >= 2.2.2)
     addressable (2.7.0)
       public_suffix (>= 2.0.2, < 5.0)
     airbrussh (1.4.0)
       sshkit (>= 1.6.1, != 1.7.0)
     annotate (3.1.1)
       activerecord (>= 3.2, < 7.0)
       rake (>= 10.4, < 14.0)
-    apipie-rails (0.5.17)
+    apipie-rails (0.5.18)
       rails (>= 4.1)
     arbre (1.2.1)
       activesupport (>= 3.0.0)
     ast (2.4.0)
-    aws-eventstream (1.0.3)
-    aws-partitions (1.289.0)
-    aws-sdk-core (3.92.0)
-      aws-eventstream (~> 1.0, >= 1.0.2)
+    aws-eventstream (1.1.0)
+    aws-partitions (1.326.0)
+    aws-sdk-core (3.98.0)
+      aws-eventstream (~> 1, >= 1.0.2)
       aws-partitions (~> 1, >= 1.239.0)
       aws-sigv4 (~> 1.1)
       jmespath (~> 1.0)
-    aws-sdk-kms (1.30.0)
+    aws-sdk-kms (1.33.0)
       aws-sdk-core (~> 3, >= 3.71.0)
       aws-sigv4 (~> 1.1)
-    aws-sdk-s3 (1.61.1)
-      aws-sdk-core (~> 3, >= 3.83.0)
+    aws-sdk-s3 (1.67.1)
+      aws-sdk-core (~> 3, >= 3.96.1)
       aws-sdk-kms (~> 1)
       aws-sigv4 (~> 1.1)
-    aws-sigv4 (1.1.1)
+    aws-sigv4 (1.1.4)
```

Comment:
### [aws-sigv4](https://github.com/aws/aws-sdk-ruby)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 1.1.1 → 1.1.4

[_[change-log](https://github.com/aws/aws-sdk-ruby/tree/master/gems/aws-sigv4/CHANGELOG.md), [source-code](https://github.com/aws/aws-sdk-ruby/tree/master/gems/aws-sigv4)_]





<details>
<summary>Commits</summary>

A change of **5** commits. See the full changes on [the compare page](https://github.com/aws/aws-sdk-ruby/compare/1.1.1...1.1.4).

These are the individual commits:
- (1cbceb7) [Version 1.1.2 of the AWS SDK for Ruby](https://github.com/aws/aws-sdk-ruby/commit/1cbceb7047c0f073958396b6a25599e086ad950c)
- (ebaa443) [Use lazy load hook to add ActionMailer delivery method](https://github.com/aws/aws-sdk-ruby/commit/ebaa4433437332f4345c101c3d660ae88a41c8c9)
- (34d2f22) [Version 1.1.3 of the AWS SDK for Ruby](https://github.com/aws/aws-sdk-ruby/commit/34d2f22c19da9b66e9cdd41a2967db8b9f08198f)
- (e421fda) [Merge commit 'ebaa4433437332f4345c101c3d660ae88a41c8c9'](https://github.com/aws/aws-sdk-ruby/commit/e421fda292edf1580d50c5812f22419c9116b883)
- (e0ea09b) [Version 1.1.4 of the AWS SDK for Ruby](https://github.com/aws/aws-sdk-ruby/commit/e0ea09b9d93f77575d8b4a93aa65a82134496dca)


</details>